core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -33,15 +33,27 @@ oppia.animation('.oppia-collection-animate-slide', function() {
 oppia.controller('CollectionPlayer', [
   '$scope', '$http', 'ReadOnlyCollectionBackendApiService',
   'CollectionObjectFactory', 'CollectionPlaythroughObjectFactory',
-  'alertsService',
+  'alertsService', 'UrlInterpolationService',
   function($scope, $http, ReadOnlyCollectionBackendApiService,
     CollectionObjectFactory, CollectionPlaythroughObjectFactory,
-    alertsService) {
+    alertsService, UrlInterpolationService) {
     $scope.collection = null;
     $scope.collectionPlaythrough = null;
     $scope.collectionId = GLOBALS.collectionId;
     $scope.showingAllExplorations = !GLOBALS.isLoggedIn;
     $scope.previewCardIsShown = true;
+    $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+    $scope.pathIconParameters = [];
+
+    $scope.setIconHighlight = function(index) {
+      document.getElementById('highlight' + index).setAttribute('class',
```

Comment:
This isn't a great way to do it; in Angular, we generally eschew direct DOM manipulation. How about maintaining a variable $scope.activeIndex or similar and adding an ng-show="..." attribute to the tag?